packages/mobile/locales/en-US/translation.json

```diff
@@ -1074,5 +1074,32 @@
     "body": "Your Total Balance is an approximate sum of all your Valora assets. Your Total Balance will change as prices fluctuate. It is updated every 10 minutes.\nSome tokens may not be shown in the Total Balance because the price is unavailable at the time of calculation.",
     "dismiss": "Dismiss"
   },
-  "personaAccountEndpointFail": "Identity verification is not currently available. Please try again later"
+  "personaAccountEndpointFail": "Identity verification is not currently available. Please try again later",
+  "linkBankAccountScreen": {
+    "tryAgain": "Try Again",
+    "completed": {
+      "title": "Verification Complete",
```

Comment:
"verification Completed"?

Reply:
fwiw I prefer "complete"

Reply:
to me, "completed" makes it sound like "verification" is the subject and "completed" is an action verb in the past tense, whereas "complete" sounds like an adjective describing "verification" as a process

Reply:
I also prefer 'Complete'